modules/monitoring-disabling-grafana.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+[id="disabling_grafana_{context}"]
+= Disabling the default Grafana deployment
+
+By default a read-only Grafana instance is deployed with a collection of dashboards displaying cluster metrics. This Grafana instance is not user-configurable.
+
+You can disable this Grafana deployment, causing the associated resources to be deleted from the cluster. You might do this if you don't need these dashboards and want to conserve resources in your cluster. You will still be able to graph metrics and view the dashboards included in the OpenShift Container Platform web console. Grafana can be safely enabled again at any time.
```

Comment:
this Grafana deployment
->
the Grafana deployment

Reply:
Fixed, but I changed the wording for the bit about the console:

> You will still be able to view metrics and dashboards included in the OpenShift Container Platform web console.

Let me know if that works.

Reply:
lgtm